.azure-pipelines/all_master.yml

```
@@ -12,6 +12,7 @@ variables:
 jobs:
 - template: './templates/test-all-checks.yml'
   parameters:
+    py2: false  # Don't run python2 tests on master
```

(nit)

```suggestion
    run_py2_tests: false
```

or `run_with_py2`, to be more explicit ?